config/config.go

```diff
@@ -200,6 +204,34 @@ type CustomChainConfig struct {
 	SDKNetwork *sdkConfig.Network `koanf:"sdk_network"`
 }
 
+// ArchiveConfig is information about the nodes for a network.
+type ArchiveConfig struct {
+	// DefaultNode is information about the node to get data from by default.
+	DefaultNode *NodeConfig `koanf:"default"`
+	// ConsensusNode is information about the node to get consensus data from,
+	// instead of the default node.
+	ConsensusNode *NodeConfig `koanf:"consensus"`
```

Comment:
might not need this, but I added just to be thorough :upside_down_face:

Reply:
I would be happy to either 
- omit it (I don't expect we'll use it, so simple = good; and we wouldn't lose expressive power, just gain verbosity in one extreme case) or
- stick it under `runtimes` (`layers`? :confounded:).

But LGTM whichever way, including leaving this as-is.